### ISSUE TITLE
Kontakte - Adressettiketten drucken

### ISSUE THREAD

**Issue by Christian-H** [yakamara/redaxo_prozer#208]
könnte so manchem helfen, aber keine hohe Prio :-)

**Comment by dergel** [comment]
gehört für mich nicht hier rein... Adressen kann man über die API exportieren und dann dafür verwenden

**Comment by Christian-H** [comment]
ja, aber nicht jeder hat das recht und das wissen die api zu benutzen. Es geht ja auch meistens nur um eine Adresse. Vielleicht ist es eine Lösung, die Postadresse in die Zwischablage per Knopfdruck zu kopieren. Dies könnte auch interessant sein, für Telefonnummern, da mit VoiP Software, ich vom PC dann aus wähle.

**Comment by frag-seb** [comment]
Zwischenablage kopieren ist nicht so einfach per möglich wie sich das anhört. 
Per Javascript sollte so was nicht möglich sein, max mit flash und ich bin gar kein freund davon!

**Comment by Christian-H** [comment]
Wir unterhalten uns gerade über eine neue Telefonanlage mit der Software 3CX und Gerd, unser IT-Dienstleister meinte, er könnte uns ein Skript (2 Zeilen) geben zum Einbau in Prozer, womit die Telefonnummer in die Zwischenablage kopiert werden kann. 
Von daher kommt die Idee, dass man die Adresse auch in die Zwischenablage kopieren könnte.

**Comment by tbaddade** [comment]
Folgender Vorschlag. Klickt man einen Kontakt an, könnte man ober-oder unterhalb von **Adresse bearbeiten** die Adresse als reinen Text ausgeben. Kopieren fertig.

**Comment by Christian-H** [comment]
Hört sich für mich gut an.
